cmd/ps.go

```diff
@@ -56,7 +56,9 @@ func PsScale(appID string, targets []string) error {
 			fmt.Printf("'%s' does not match the pattern 'type=num', ex: web=2\n", target)
```

Comment:
yeah but what I mean is that if there is a single bad scale type we should error and say "hey, this process type is poorly formed, so we're not going to accept this request."

Same thing applies for `config:set` as well. For example there were issues like https://github.com/deis/workflow-cli/issues/61 that was unintended, so there is precedence.